modules/ROOT/pages/inline-css.adoc

```diff
@@ -0,0 +1,116 @@
+
+== Interactive example
+
+////
+  No Lorem Ipsum! Add some content to test on, possibly just a description of how end-users use the feature?
+  See the emoticons, advtable,and advcode demos for examples.
+////
+
+liveDemo::inline-css[]
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: '{plugincode}',
+});
+----
+
+== Usage notes
+
+// Document what we are and are not supporting. Set out the specific
+// limitations of this plugin and its API.
+// 
+// 1. We skip pseudo-elements and most @-rules.
+// 
+// 2. Vogue only supports `@import` @-rules at this stage.
+// 
+// See the Jira ticket: https://ephocks.atlassian.net/browse/TINY-9265
+// 
+// And see the tech spec for @-rule support:
+// https://notion.so/tinycloud/Technical-specification-7c16d4522d1241ba9e7364880eab3e51#555fede3a4ff4a26ad5efdf72bc1af6f.
+
+// This plugin allows extracting the content of the editor with CSS inlined.
+// Support classic and inline modes. Can be used in shadow dom. 
+// Should be able to extract CSS from the stylesheets defined via
+// `content_css` and `font_css` options, and CSS styles defined via `style_css` option.
+// 
+// Does not support:
+// * Pseudo-classes ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes))
+//   - Example: `button:hover { color: blue; }`
+// * Pseudo-elements ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements))
+//   - Example: `p::first-line { color: blue; }`
+// * At rules ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/At-rule))
+//	  - Example: @media (screen)
+// * !important property
+//   
+
+// Does support: 
+// * @import with no at-rules at this stage
+// * some pseudo-class selectors. See the specification for more details
+// (https://notion.so/tinycloud/Technical-specification-7c16d4522d1241ba9e7364880eab3e51#94b1d13ff444492bbfce789f7876f39c)
+
+// There is only one API: `getContent()`.
+// Regarding to the events, it dispatches `InlineCSS` event.
+// I think it is OK to document this event even though its purpose isn’t really for the user’s usage.
+
+
+== Options
+
+The following configuration options affect the behavior of the {pluginname} plugin.
+
+include::partial$configuration/inline-css.adoc[]
+
+// ‘The target group is developers, not end-users’: <https://ephocks.atlassian.net/browse/EPIC-31>
+// Consequently, I’ve assumed the Inline CSS plugin will not be invoked by:
+// * using a Toolbar button; nor by
+// * selecting from a menu.
+
+== Commands
+
+// I’ve assumed — in semi-blind fashion — that there is a
+// `mceInlineCSS` command.
+
+The {pluginname} plugin provides the following {productname} command.
+
+include::partial$commands/{plugincode}-cmds.adoc[]
+
+== Events
+
+// I’ve assumed — still in semi-blind fashion — that there are 
```

Comment:
There is one event: `InlineCSS` that is fired *at the start* of inlining the css.